libclc/clc/lib/clspv64

```diff
@@ -0,0 +1 @@
+clspv
```

Comment:
I don't understand this bit. What's happening here, if you please?

Reply:
This might be github being github, but `libclc/clc/lib/clspv64` is a symlink to `libclc/clc/lib/clspv` in the same way that `libclc/clspv64` is already a symlink to `libclc/clspv`